assets/js/components/GoogleChart/index.js

```diff
@@ -258,6 +258,8 @@ export default function GoogleChart( props ) {
 			return;
 		}
 
+		let hasMovedLegend = false;
```

Comment:
Is this necessary? The end result is a mutation of the style which should be idempotent, no?

Reply:
Oh, shoot, you're totally right. I had a brain fail and thought I was subtracting ten each time—so I thought it would mess up with multiple markers. 🤦🏻 

I'll remove 👍🏻